spec/lib/environment_config_transmogrifier_spec.rb

```diff
@@ -63,6 +64,31 @@
       expect(new_config['properties']['parent_key']['child_key']['grandchild_key']).to eq 'bar'
     end
 
+    it 'should error on triple-paren' do
+      # Arrange
+      environment_templates = {
+        'properties.parent_key.child_key.grandchild_key' => 'f(((MY_FOO_VAR)))'
+      }
+      # Act
+      # Assert
+      expect {
+        EnvironmentConfigTransmogrifier.transmogrify(@base_config, environment_templates)
+      }.to(raise_exception(LoadYamlFromMustacheError,
+                           /Could not load config key.*Illegal content in tag/))
+    end
+
+    it 'should support changing templates inline' do
+      # Arrange
+      environment_templates = {
+        'properties.parent_key.child_key.grandchild_key' => '((={{ }}=))f({{MY_FOO_VAR}})'
```

Comment:
This is… less than ideal.  Does `'(("("))((MY_FOO_VAR))((")"))'` work?  It's just a _different_ type of ugly, though, and not actually _less_ ugly :|